guides/common/modules/ref_dns-options-for-network-configuration.adoc

```diff
@@ -2,28 +2,37 @@
 = DNS Options for Network Configuration
 
 --foreman-proxy-dns::
-  Enables DNS service.
+  Enables the DNS feature.
 You can set this option to `true` or `false`.
 
+--foreman-proxy-dns-provider::
+  Selects the provider to be used.
+
 --foreman-proxy-dns-managed::
-  Enables Foreman to manage the DNS service.
+  Let the installer manage ISC BIND.
+This is only relevant when using the `nsupdate` and `nsupdate_gss` providers.
 You can set this option to `true` or `false`.
 
 --foreman-proxy-dns-forwarders::
   Sets the DNS forwarders.
-Set this to your DNS servers.
+Only used when DNS is managed.
```

Comment:
```suggestion
Only used when ISC BIND is managed by the installer.
```

I stumbled over this while reading, and wanted to hear your thoughts.
My reasoning is: DNS (as in the entries) are "always" managed by Foreman if the Proxy has the DNS feature. Which DNS *backend* is used and whether this backend is *also* configured by the Installer, is a completely different question.

For `--foreman-proxy-dns-managed` you have written "Let the installer manage ISC BIND." and this is correct, as this option toggles whether the installer installs and configures BIND for us.
Most of the later `--foreman-proxy-dns-*` options are only relevant if the above is true, but I don't think "when DNS is managed" gives the reader the right understanding of the situation.